packages/fluent-theme/src/fluent/styles/Toggle.styles.ts

```diff
@@ -1,14 +1,52 @@
-export const ToggleStyles = {
-  pill: {
-    width: '40px',
-    height: '20px',
-    borderRadius: '10px',
-    padding: '0 4px'
-  },
-  thumb: {
-    width: '12px',
-    height: '12px',
-    borderRadius: '12px',
-    borderColor: 'transparent'
+import { IToggleProps } from '../../../../office-ui-fabric-react/lib';
```

Comment:
this import should be `import { IToggleProps } from 'office-ui-fabric-react/lib/Toggle'`... I believe you used the auto import feature which is sometimes buggy))

Reply:
Thanks for catching that, I'll update it. I love the auto import feature... 😢